tests/installation/start_install.pm

```diff
@@ -134,4 +134,17 @@ sub run {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    if (check_var('UPGRADE', '1')) {
+        select_console 'log-console';
+        save_screenshot;
+        script_run "save_y2logs /tmp/y2logs.tar.bz2";
+        upload_logs "/tmp/y2logs.tar.bz2";
```

Comment:
I think this need 'set_var('Y2LOGS_UPLOADED', 1);' to avoid duplicate upload.

Reply:
already added at line 145:  set_var('Y2LOGS_UPLOADED', 1);

Reply:
ok